src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -430,55 +430,32 @@ describe('non-empty successful affected clusters table', () => {
     ).forEach(([category, label]) => {
       SORTING_ORDERS.forEach((order) => {
         it(`${order} by ${label}`, () => {
-          const col = `td[data-label="${label}"]`;
-          const header = `th[data-label="${label}"]`;
-
-          cy.get(col).should(
-            'have.length',
-            Math.min(DEFAULT_ROW_COUNT, data.length)
-          );
-          if (order === 'ascending') {
-            cy.get(header).find('button').click();
-          } else {
-            cy.get(header).find('button').dblclick();
-          }
-
-          // add property name to clusters
-          let sortedClusters = _.cloneDeep(dataUnsorted);
-          // convert N/A timestamps as really old ones
-          sortedClusters.forEach((it) => {
-            if (it['last_checked_at'] === '') {
-              it['last_checked_at'] = '1970-01-01T01:00:00.001Z';
-            }
-            if (it.meta.cluster_version === '') {
-              it.meta.cluster_version = '0.0.0';
-            }
-          });
-
+          let sortingParameter = category;
+          // modify sortingParameters for certain values
           if (category === 'name') {
             // name sorting is case insensitive
-            category = (it) => it.name.toLowerCase();
+            sortingParameter = (it) => it.name.toLowerCase();
+          } else if (category === 'last_checked_at') {
```

Comment:
the object is `filtersConf` and it has nothing to do with `sorting`. That's why I do not even though about it. We can create a more general object than encloses both, filters and sorting utilities, but... we can also add other stuff for other tests and it can blow up. I am not against the idea, but we should be cautious with it.

Note also that there is no 1-1 map between filters and columns. E.g. in clusters page,we can sort by a particular risk field (e.g. Critical), but the filter is risk and critical an option within it.

Reply:
ok thanks for the explanation :)